packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -240,13 +240,6 @@ export const messages = {
   isv_debug_bootstrap_generate_launchjson: 'Creating launch configuration',
   isv_debug_bootstrap_open_project: 'Opening project in Visual Studio Code',
 
-  force_apex_log_get_text: 'SFDX: Get Apex Debug Logs...',
-  force_apex_log_get_no_logs_text: 'No Apex debug logs were found',
-  force_apex_log_get_pick_log_text: 'Pick an Apex debug log to get',
-  force_apex_log_list_text: 'Getting Apex debug logs',
-  apex_log_get_text: 'Get Apex Logs',
-  apex_log_list_text: 'Fetching Apex debug log list',
```

Comment:
I removed these messages so both versions of the command use the same messages. This is related to a discussion we'll have in parking lot on 2/5/21